sdks/builds/wasm.mk

```diff
@@ -103,6 +103,8 @@ _wasm-$(1)_CONFIGURE_FLAGS= \
 	--enable-maintainer-mode \
 	--enable-minimal=appdomains,com,remoting \
 	--enable-icall-symbol-map \
+	--with-cooperative-gc=no \
+	--enable-hybrid-suspend=no \
```

Comment:
is there a problem around safepoints?

Reply:
the device runtime is compiled to not use them, so its missing the relevant icall so it asserts when loading aot-ed code.